Sources/SwiftSDKTool/InstallSwiftSDK.swift

```diff
@@ -47,7 +49,10 @@ public struct InstallSwiftSDK: SwiftSDKSubcommand {
             swiftSDKsDirectory: swiftSDKsDirectory,
             fileSystem: self.fileSystem,
             observabilityScope: observabilityScope,
-            outputHandler: { print($0.description) }
+            outputHandler: { print($0.description) },
+            downloadProgressAnimation: ThrottledProgressAnimation(
```

Comment:
Progress animation is throttled by timer to avoid too flickering animation triggered by `HTTPClient`'s too frequent progress reports.

Reply:
Since `swift-tools-support-core` is deprecated, would you mind adding this new types to the `Basics` module in SwiftPM instead?

Reply:
Okay, move it to SwiftPM repo